packages/astro/tsconfigs/base.json

```diff
@@ -5,11 +5,11 @@
     "target": "ESNext",
     "module": "ESNext",
     // Enable node-style module resolution, for things like npm package imports.
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
+    // Allow importing TypeScript files using their native extension (.ts(x)).
+    "allowImportingTsExtensions": true,
     // Enable JSON imports.
     "resolveJsonModule": true,
-    // Enable stricter transpilation for better output.
-    "isolatedModules": true,
```

Comment:
Why is this no longer needed?

Reply:
You can't have `isolatedModules` and `verbatimModuleSyntax` on at the same time, but I guess since they're in separate preset we could enable it and disable it in strict, maybe. Or maybe `verbatimModuleSyntax` should be part of base anyways...

Reply:
Oh, it seems like they changed it https://github.com/microsoft/TypeScript/pull/53611